README.md

```diff
@@ -32,6 +32,23 @@ use a major version when expressing a dependency on `aws-sdk`.
 gem 'aws-sdk', '~> 2'
 ```
 
+##Getting Help
+
+Please use these community resources for getting help. We use the GitHub issues for tracking bugs and feature requests and have limited bandwidth to address them.
+
+* Ask a question on StackOverflow and tag it with `aws-sdk-ruby`
+* Come join the AWS Ruby community chat on gitter: `aws/aws-sdk-ruby`
```

Comment:
Let's go ahead and link it - for example:

`* Come join the AWS SDK for Ruby [Gitter Channel](https://gitter.im/aws/aws-sdk-ruby)`